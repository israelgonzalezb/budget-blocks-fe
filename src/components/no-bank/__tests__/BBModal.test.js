
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import BBModal from '../BBModal';


describe('BBModal test', () => {
  it('BBModal should match snapshot', () => {
    const component = renderer.create(<BBModal
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // hey there...
  });
});
