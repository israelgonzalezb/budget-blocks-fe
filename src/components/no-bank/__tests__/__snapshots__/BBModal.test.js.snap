// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BBModal test BBModal should match snapshot 1`] = `null`;
